.github/workflows/pr_helper.yml

```diff
@@ -16,39 +17,41 @@ jobs:
       pull-requests: write
 
     steps:
-      - uses: actions/checkout@v4
-      - name: Setup python environment
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Python environment
         uses: actions/setup-python@v5
         with:
           python-version: 3.8
           cache: pip
-          cache-dependency-path: |
-            infra/ci/requirements.txt
 
-      - name: Install dependencies
+      - name: Install Python dependencies
         run: |
-          python -m pip install --upgrade pip
+          pip install --upgrade pip
           pip install -r infra/ci/requirements.txt
 
-      - name: setup go environment
+      - name: Setup Go environment
         uses: actions/setup-go@v5
         with:
-          go-version: 'stable'
-      - run: go install github.com/ossf/criticality_score/cmd/criticality_score@latest
+          go-version: stable
+
+      - name: Install Criticality Score tool
+        run: go install github.com/ossf/criticality_score/cmd/criticality_score@latest
 
-      - name: Check if authors are authorized to modify.
-        id: checkAuthor
+      - name: Check if authors are authorized to modify
+        id: check-author
         env:
-          GITHUBTOKEN: ${{secrets.GITHUB_TOKEN}}
-          PRAUTHOR: ${{ github.event.pull_request.user.login }}
-          PRNUMBER: ${{ github.event.pull_request.number }} 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
         run: python infra/pr_helper.py
```

Comment:
Remove this please. The Python code uses these env vars.

Reply:
Hi @jonathanmetzman ,

I want to contribute to the project but I'm a beginner in open source. Could you guide me on how to approach and make valuable contributions?